tools/rpkg/tests/testthat/_snaps/types.new.md

```diff
@@ -0,0 +1,237 @@
+# test_all_types() output
```

Comment:
These files aren't meant to be committed. But I also typically don't ignore them because their presence indicates trouble.

You can run `testthat::snapshot_accept()` (to see why the difference occurs) and then restore the `tests/` directory. I don't have a good solution for the problem at hand.